include/gz/utils/ExtraTestMacros.hh

```diff
@@ -25,14 +25,24 @@
 /// as part of the test suite execution.
 /// The macro uses the Disabled_ prefix provided by googletest. See
 /// https://chromium.googlesource.com/external/github.com/google/googletest/+/HEAD/googletest/docs/advanced.md
+/// TODO(gz-utils4): remove the windows alias
 #define GZ_UTILS_TEST_ENABLED_ONLY_ON_WINDOWS(TestName) \
-  DETAIL_GZ_UTILS_TEST_ENABLED_ONLY_ON_WINDOWS(TestName)
+  DETAIL_GZ_UTILS_TEST_ENABLED_ONLY_ON_WIN32(TestName)
+
+/// Other platforms will get the test compiled but it won't be run
+/// \brief Restrict the execution of the test to just the Windows platform
+/// as part of the test suite execution.
+/// The macro uses the Disabled_ prefix provided by googletest. See
+/// https://chromium.googlesource.com/external/github.com/google/googletest/+/HEAD/googletest/docs/advanced.md
+#define GZ_UTILS_TEST_ENABLED_ONLY_ON_WIN32(TestName) \
+  DETAIL_GZ_UTILS_TEST_ENABLED_ONLY_ON_WIN32(TestName)
 
 /// \brief Restrict the execution of the test for the Windows platform.
 /// The test will be compiled on Windows too but will never be run as
 /// part of the test suite. The macro uses the Disabled_ prefix provided
 /// by googletest. See
 /// https://chromium.googlesource.com/external/github.com/google/googletest/+/HEAD/googletest/docs/advanced.md
+/// TODO(gz-utils4): remove the win32 alias
```

Comment:
Curious, why do we need to remove the win32 alias? Your above comment says that we want to remove the windows alias, wouldn't that be sufficient?

Reply:
Removed in 9a4fcac